phoenicis-repository/src/main/java/org/phoenicis/repository/repositoryTypes/ClasspathRepository.java

```diff
@@ -76,6 +79,11 @@ public RepositoryDTO fetchInstallableApplications() {
         }
     }
 
+    @Override
+    public boolean isSafe() {
```

Comment:
I don't know, but I know that the user should have the choice to decide, when he wants to add a repository, if the repository is "safe" or not, right?